### PR TITLE
Lesson001

### PR DESCRIPTION
**Pull request by AnMatveeva**

### CODE REVIEW

File: lesson001/task1/lesson1-1.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html leng="ru">
+    <head>
+        <meta charset="utf-8" />
+        <title>HTML</title>
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <p>
+            Язык <b>HTML</b> интерпретируется <u>браузерами</u>; полученный в результате интерпретации <br />
```
Comment (Onix-one) [inline lesson001/task1/lesson1-1.html]: браузерами;     <---    mast be link. in original text its link.

File: lesson001/task1/lesson1-1.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            форматированный текст отображается на экране монитора компьютера или мобильного устройства. <br />
+            Язык <i>HTML</i> до 5-ой версии определялся как приложение <u>SGM</u> (стандартного обобщённого языка <br />
+            разметки по  стандарту <u>ISO</u> 8879). Спецификации HTML5   формулируются в <br />
+            терминах <u><i>DOM</i></u> (объектной модели документа).  <br />
+            Язык <u><i>XHTML</i></u> является более строгим вариантом HTML, он следует синтаксису <u>XML</u> и является <br />
+            приложением языка XML в области разметки гипертекста. <br />
+            Во всемирной паутине HTML-страницы, как правило, передаются браузерам от сервера по <br />
+            протоколам <u><i>HTTP</i></u> или <u><i>HTTPS</i></u>, в виде простого текста или с использованием <u>шифрования</u>.
```
Comment (Onix-one) [inline lesson001/task1/lesson1-1.html]: all underlined text must to be as a link

File: lesson001/task2/lesson2-2.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+         <td colspan="2"></td>
+         
+         <td rowspan="3"></td>
+         <td></td>
+        </tr>
+        <tr>
+         
+         <td colspan="2"></td>
+         
+         <td rowspan="2"></td>
```
Comment (Onix-one) [inline lesson001/task2/lesson2-2.html]: format code plz. need toremove empty strings

File: lesson001/task2/lesson2-3.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <meta charset="utf-8" />
+        <title></title>
+    </head>
+    <body>
+        <h4>Nested lists</h4>
+        <ul>
+            <li>Monday</li>
+        <ol>
+            <li>First day of the week</li>
+            <li>Hard day</li>
```
Comment (Onix-one) [inline lesson001/task2/lesson2-3.html:13]: )))) you made me smile )))))

File: lesson001/task3/lesson3-1.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                color: black;
+                padding: 8px;
+            }
+            body {
+                color: rgb(0, 0, 0);
+            }
+            p {
+                font-style: italic;
+                color: rgb(67, 56, 43);
+            }
```
Comment (Onix-one) [inline lesson001/task3/lesson3-1.html]: Move all this code to a file ---> style.css. Putting styles here and in tags is bad form. Styles are usually applied through classes.

File: lesson001/task3/lesson3-1.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        </style>
+        <link href="styles/style.css" rel="stylesheet" >
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <p>
+            <font color="#716F3B" size="4">Язык <b>HTML</b></font> интерпретируется <u>браузерами</u>; полученный в результате интерпретации <br />
```
Comment (Onix-one) [inline lesson001/task3/lesson3-1.html]: The same....  color="#716F3B" use styles through clases css from file style.css

File: lesson001/task3/lesson3-2-3.html [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+                border: 6px outset rgb(0, 255, 94);
+            }
+            .p6{
+             border: 30px solid transparent;
+             border-image: url(1613801046_67-p-zolotistie-ramki-na-fone-89.png) 150 round round;
+             width: 500px;
+             height: 60px;
+             background-color: mistyrose;
+            }
+        </style>
```
Comment (Onix-one) [inline lesson001/task3/lesson3-2-3.html]: Same situation.  and  Borders does not match the task

File: lesson001/task1/lesson1-1.html (hunk 1 above)
Comment (AnMatveeva) [reply lesson001/task1/lesson1-1.html]: Completed but no links displayed in source text

File: lesson001/task1/lesson1-1.html (hunk 2 above)
Comment (AnMatveeva) [reply lesson001/task1/lesson1-1.html]: Completed

File: lesson001/task2/lesson2-2.html (hunk 3 above)
Comment (AnMatveeva) [reply lesson001/task2/lesson2-2.html]: Completed

File: lesson001/task2/lesson2-3.html (hunk 4 above)
Comment (AnMatveeva) [reply lesson001/task2/lesson2-3.html:13]: )

File: lesson001/task3/lesson3-1.html (hunk 5 above)
Comment (AnMatveeva) [reply lesson001/task3/lesson3-1.html]: Completed

File: lesson001/task3/lesson3-1.html (hunk 6 above)
Comment (AnMatveeva) [reply lesson001/task3/lesson3-1.html]: Completed

File: lesson001/task3/lesson3-2-3.html (hunk 7 above)
Comment (AnMatveeva) [reply lesson001/task3/lesson3-2-3.html]: Completed

File: lesson001/task1/lesson1-1.html [hunk 8]
```diff
@@ -7,15 +7,15 @@
     <body>
         <h1>HTML</h1>
         <p>
-            Язык <b>HTML</b> интерпретируется <u>браузерами</u>; полученный в результате интерпретации <br />
+            Язык <b>HTML</b> интерпретируется <a href="browser.html"><u>браузерами</u></a>; полученный в результате интерпретации <br />
             форматированный текст отображается на экране монитора компьютера или мобильного устройства. <br />
```
Comment (Onix-one) [inline lesson001/task1/lesson1-1.html]: )))) Here must be another link. If you click on the link in the original document, you will be taken to the Wikipedia article. and in file with logic markup must be links too.

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: lesson001/task1/lesson1-1.html (hunk 8 above)
Comment (AnMatveeva) [reply lesson001/task1/lesson1-1.html]: Completed

**Pull request merged**